src/assets/font-awesome-custom/svg/miscs/drill.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="958.6" height="958.6"><path d="M2 946l-2 2v1l2 2v4h3v-2-1a2 2 0 010-1h4v-5zm7 1v4l2-2h3v-1h-3l-2-1zm-6 1l-1 1c-1 0-1 0 0 0l1-1zm2 0l-1 1H3l1-1h1zm-3 8H1v2l1 1h5v-3H2z"/></svg>
```

Comment:
You need to stretch your icon in a 14x14  box. As now it's in a 958x958, so everything will be out of the visible area.

Good thing to know, it will be displayed in exatly 14x14 pixels, so you can work on having a pixel perfect icon.